client/modules/_hooks/src/workspace/useGetJobs.ts

```diff
@@ -38,6 +38,23 @@ export function useGetJobs(
   return useQuery(getJobsQuery(operation, queryParams));
 }
 
+const getJobQuery = (
+  operation: TJobGetOperations,
+  queryParams: TJobParamsType
+) => ({
+  queryKey: ['workspace', 'getJobs', operation, queryParams],
+  queryFn: () => getJobs(operation, queryParams),
+  retry: false,
+  enabled: !!queryParams.uuid,
+});
+
+export function useGetJob(
```

Comment:
Is this a fundamentally different request than `useGetJobs`?

Reply:
the difference is in query, this uses getJobQuery which is disabled if uuid is not available. Whereas, useGetJobs proceeds with an empty uuid and retrieves the full list. We do not want to retrieve full list in this specific usecase. The goal is to only go through with the request if uuid.

Reply:
And I cannot do conditional (!!uuid) retrieval with useGetJobs in the component - react does not like it.